spec/system/candidate_entering_contact_details_spec.rb

```diff
@@ -35,9 +35,10 @@
       click_on t('application_form.save_and_continue')
     end
 
-    it 'sees an error summary' do
+    it 'sees an error summary with clickable links', js: true do
```

Comment:
I guess we hope to avoid repeating this test for each screen! In this case it drives out the generalisation so it's useful — what's the right amount of coverage once this branch is merged, do you think?

Reply:
Good question! I quite like testing it at this level rather than just testing the error summary component in isolation, but repeating it in every spec could be tedious. 

I've got mixed feelings on the rspec abstractions - `behaves like` and similar - for this level of spec, but that's another option.

One to think about as we go, I think?